src/components/IconWrapper/IconWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from './IconWrapper.module.css';
+
+interface IIconWrapper {}
+
+/**
+ * Icon Wrapper aims to set minimal width to icons and stop them disapearing with resizing.
+ *
+ * @param children - icon to display
+ */
+export const IconWrapper = ({ children }: React.PropsWithChildren<IIconWrapper>) => {
+  return (
+    <span className={styles['icon-wrapper']}>
+      <>{children}</>
+    </span>
+  );
```

Comment:
- `return` statement is not needed (`() => <span>...</span>` syntax is more suitable here)
- Import `PropsWithChildren` directly
- Use `{children}`(without the `<></>`)

Reply:
Thank you